src/store/entitiesReducer.js

```diff
@@ -32,23 +31,28 @@ export const displayState = (data) => {
     return (<SystemState stateValue={ data }/>);
 };
 
-export const scoreProgress = () => (data) => {
+export const scoreProgress = () => (data, id, { state }) => {
     return (
-        <ProgressScoreBar measureLocation='outside'
-            utilizedValue={data} />
+        state === NO_DATA_STATE ?
+            <span>{ NO_DATA_VALUE }</span> :
+            <ProgressScoreBar measureLocation='outside'
+                utilizedValue={data} />
     );
 };
 
 export const recommendations = (data, id, { inventory_id: inventoryId, isDeleted, state }) => {
     return (
-        (isDeleted || state === NO_DATA_STATE)
-            ? <span className='recommendations'>{ data }</span>
-            : (
-                <Link to={{ pathname: `/${inventoryId}` }}
-                    className={ `pf-link recommendations link-${inventoryId}` }>
-                    { data }
-                </Link>
-            )
+        isDeleted ? <span className='recommendations'>{ state === NO_DATA_STATE ? NO_DATA_VALUE : data }</span>
+            : state === NO_DATA_STATE ?
+                (
+                    <span>{ NO_DATA_VALUE }</span>
```

Comment:
@PreetiW - It seems that with `recommendations` CSS class [here](https://github.com/RedHatInsights/ros-frontend/blob/main/src/Routes/RosPage/ros-page.scss#L25), it adjusts the left padding.
One observation: with this change, when there will be multiple records, vertical alignment for column `Suggestions` will not be same for rows.